barretenberg/cpp/src/barretenberg/vm2/constraining/recursion/goblin_avm_recursive_verifier.hpp

```diff
@@ -135,7 +132,7 @@ class AvmGoblinRecursiveVerifier {
         MegaRecursiveVerifier mega_verifier(&ultra_builder, mega_vk);
         StdlibProof<UltraBuilder> mega_proof =
             bb::convert_native_proof_to_stdlib(&ultra_builder, inner_output.mega_proof);
-        auto mega_verifier_output = mega_verifier.verify_proof(mega_proof, input_points_accumulator);
+        auto mega_verifier_output = mega_verifier.verify_proof(mega_proof);
```

Comment:
I looks like we're missing the aggregation of `mega_verifier_output.points_accumulator` with `goblin_verifier_output.pairing_points` (which seems to just be ignored altogether since we only return the ipa_claim/proof below)

Reply:
Kind of a bummer that something like this can happen without something breaking but its tricky to see how to enforce this more strictly. The consequences of missing it are pretty dire though..